elixir/message/mix.exs

```diff
@@ -0,0 +1,31 @@
+defmodule Message.MixProject do
+  use Mix.Project
+
+  def project do
+    [
+      app: :message,
+      version: "0.1.0",
+      elixir: "~> 1.12",
+      start_permanent: Mix.env() == :prod,
+      deps: deps()
+    ]
+  end
+
+  # Run "mix help compile.app" to learn about applications.
+  def application do
+    [
+      extra_applications: [:logger],
+      mod: {Message.Application, []}
+    ]
+  end
+
+  # Run "mix help deps" to learn about dependencies.
+  defp deps do
+    [
+      {:plug_cowboy, "~> 2.5"},
+      {:opentelemetry_honeycomb, "~> 0.5.0-rc.1"},
```

Comment:
In general, we aim to move away from OTel-to-Honey-API exporters and push towards straight OpenTelemetry and "just use OTLP." Is this something that is worth demonstrating for an interim period while OTel + OTLP + Elixir stabilizes?

Reply:
Yeah, I was of two minds about it. Was just kind of aiming for variety for the sake of illustration, but it might send the wrong message. 😅 Would be easy enough to convert it to boring old OTLP, though.